### ISSUE TITLE
[5.2.15] Cloned block not available in template

### ISSUE THREAD

**Issue by mfell** [spicywebau/craft-neo#961]
### Bug Description

A cloned block does not appear in the frontend. 

Screen capture: http://files.fell.hamburg/Craft5Neo5CloneFE_updated.mov

Please let me know if there’s anything further I can do to assist!

### Steps to reproduce

1. create a block with two children
2. save entry
3. open frontend -> everything is fine until now
4. clone the block
5. save entry
6. open frontend -> the cloned block is not there

### Expected behaviour

Just to see the cloned block in the frontend and ability to edit the cloned block.

### Neo version

5.2.15

### Craft CMS version

5.4.10.1

### What is the affected Neo field's propagation method?

Only save blocks to the site they were created in

### Does this issue involve templating, and if so, is eager-loading used?

This is a templating issue, no eager-loading

**Comment by mfell** [comment]
I tried investigating the template, but I’m unsure how to dig deeper than this:
```
    {% set level1Blocks = entry.contentBuilder.level(1).all() %}
    {% for block in level1Blocks %}
        {% dump(block.id) %}
    {% endfor %}
```
    
The source block is visible, another created block aswell, but the cloned block in not available in the dump.

**Comment by mfell** [comment]
I tested the issue again in both environments:

- **Craft 4.12.9** / **Neo 4.2.19** -> no issues; everything seems to be working perfectly
- **Craft 5.4.10.1** / **Neo 5.2.15** -> the bug occurs as described here

The templates are identical… Version 5 is simply an update of Version 4.

**Comment by ttempleton** [comment]
The draft state of the block is not being dropped, not sure why, but looking into it.

**Comment by mfell** [comment]
Thanks for the responds ... I'm not in hurry but happy if this issue is fixed.

**Comment by mfell** [comment]
May I kindly ask, if there is anything new for this issue?

Thank for your support!

**Comment by nikolenko-dmitriy** [comment]
Having the same trouble.

**Neo version**
5.2.15

**Craft CMS version**
5.5.0.1

**Comment by ttempleton** [comment]
Fixed now in Neo 5.2.18. I think you should be able to re-clone the affected blocks to get them appearing in your templates, but let me know if that doesn't work.

**Comment by mfell** [comment]
Yes ... seems to be working correct now ... thanks for your support!
